src/types.ts

```diff
@@ -60,3 +64,9 @@ export interface PackageJson {
     node?: string;
   };
 }
+
+export type RollupOptionsWithOutput = RollupOptions & { output: OutputOptions };
+
+export interface TSDXConfig {
+  rollup: (config: RollupOptions, options: TSDXOptions) => RollupOptions;
+}
```

Comment:
This still has the same problem as in https://github.com/formium/tsdx/pull/823#discussion_r503081245 . Should probably try being more specific here and in `createRollupConfig.ts`.